cypress/integration/homeFeedFlows/feedNavigation.spec.js

```diff
@@ -25,18 +25,12 @@ describe('Home Feed Navigation', () => {
         'aria-current',
         'page',
       );
-
-      cy.get('@week').should('have.attr', 'aria-current', '');
-      cy.get('@month').should('have.attr', 'aria-current', '');
-      cy.get('@year').should('have.attr', 'aria-current', '');
-      cy.get('@infinity').should('have.attr', 'aria-current', '');
```

Comment:
nitpick (blocking) - these assertions were needed to make sure that `aria-current` is correctly updated. It's awesome if the `aria-current` attribute just isn't there any more, but we'll need to update the tests to check that instead.

I think it's just a case of replacing these lines (and all the similar ones below) with e.g.
`cy.get('@week').should('not.have.attr', 'aria-current');`

Reply:
@aitchiss hello, thanks for your review, it's not an error, it's an optimization for https://github.com/forem/forem/issues/13289. I've added the requested changes.